python/ray/dataframe/utils.py

```diff
@@ -282,3 +282,60 @@ def decorator(cls):
         return cls
 
     return decorator
+
+
+def _map(func, *objects):
```

Comment:
I thought it would go well with the `_reduce()` implementation for generic objects in the object store. I can remove it and change back to `_map_partitions()` instead.